server/middleware.go

```diff
@@ -42,6 +43,49 @@ func JSONToHTTP(ep JSONEndpoint) http.Handler {
 	})
 }
 
+// ContextToHTTP is middleware func to convert ContextHandler to
+// an http.HandlerFunc.
+func ContextToHTTP(ctx context.Context, ep ContextHandler) http.Handler {
```

Comment:
"ContextToHTTP is **a**  middleware func to convert **a** ContextHandler an http.**Handler**"